src/cleaner/test/job/cleaner-test-job.json

```diff
@@ -1,6 +1,6 @@
 {
   "jobName": "cleaner-test-job",
-  "image": "ubuntu",
+  "image": "openpai/testcleaner",
```

Comment:
add a tag might be better

Reply:
Then I will use openpai/testcleaner:stable